zookeeper/source/zk_nn_client.cc

```diff
@@ -100,7 +100,30 @@ namespace zkclient{
 		memcpy(&data[0], znode_data, sizeof(*znode_data));
 	}
 
+	bool ZkNnClient::previousBlockComplete(uint64_t prev_id) {
+		int error_code;
+		std::vector<std::string> children;
+		std::string block_id_str = std::to_string(prev_id);
+		if (zk->get_children("/block_locations/" + block_id_str, children, error_code)) {
+			if (children.size() >= 1){
```

Comment:
We should be reading MIN_REPLICATION from a config, but for now, can you just make a constant instead of using 1?